static/scss/partials/header.scss

```diff
@@ -39,6 +39,14 @@ header {
     white-space: nowrap;
 }
 
+.sign-up {
```

Comment:
I recommend showing the sign-in button on the mid-size screens if there's room. 

Caveat: If Eduardo is good with it as is, please disregard this recommendation. 

<img width="717" alt="image" src="https://user-images.githubusercontent.com/2692333/128419352-fec2c11d-032d-440a-85e0-1662366c747e.png">